package.json

```diff
@@ -251,6 +251,12 @@
     "@metamask/network-controller@npm:^19.0.0": "patch:@metamask/network-controller@npm%3A21.0.0#~/.yarn/patches/@metamask-network-controller-npm-21.0.0-559aa8e395.patch",
     "@metamask/network-controller@npm:^20.0.0": "patch:@metamask/network-controller@npm%3A21.0.0#~/.yarn/patches/@metamask-network-controller-npm-21.0.0-559aa8e395.patch",
     "path-to-regexp": "1.9.0",
+    "@metamask/keyring-api@^8.1.3": "npm:@metamask/keyring-api@10.1.0",
+    "@metamask/keyring-api@^8.0.0": "npm:@metamask/keyring-api@10.1.0",
+    "@metamask/keyring-api@9.0.0": "npm:@metamask/keyring-api@10.1.0",
+    "@metamask/eth-snap-keyring@^4.4.0": "npm:@metamask/eth-snap-keyring@5.0.1",
+    "@metamask/eth-snap-keyring@^4.3.1": "npm:@metamask/eth-snap-keyring@5.0.1",
+    "@metamask/eth-snap-keyring@^4.3.6": "npm:@metamask/eth-snap-keyring@5.0.1",
```

Comment:
We should not merge those resolutions into the mainline.

We should update the actual packages/versions accordingly.

Reply:
They have been updated [here](https://github.com/MetaMask/metamask-extension/blob/e00721ca5532193fc6dfb32fee535c3d4667227d/package.json#L316) and [here](https://github.com/MetaMask/metamask-extension/blob/e00721ca5532193fc6dfb32fee535c3d4667227d/package.json#L327) , but without these resolutions it's not sufficient since these versions are still in use.